recognition/include/pcl/recognition/implicit_shape_model.h

```diff
@@ -597,16 +596,16 @@ namespace pcl
         std::vector<float> training_sigmas_;
 
         /** \brief This value is used for the simplification. It sets the size of grid bin. */
-        float sampling_size_;
+        float sampling_size_{0.1f};
 
         /** \brief Stores the feature estimator. */
         typename Feature::Ptr feature_estimator_;
 
         /** \brief Number of clusters, is used for clustering descriptors during the training. */
-        unsigned int number_of_clusters_;
+        unsigned int number_of_clusters_{184};
```

Comment:
Should be 0?

You haven't changed implicit_shape_model.hpp - is this for later? Would be nice to have them in pairs :smile:

Reply:
Weird omission by `clang-tidy`